app/src/main/res/resource/values-de/strings.xml

```diff
@@ -223,21 +225,21 @@
     <string name="action_align_right">Rechts anheften</string>
 
     <string name="action_wallpaper">Hintergrundbild</string>
-    <string name="action_lockscreen">LockScreen</string>
+    <string name="action_lockscreen">Sperrbildschirm</string>
     <string name="action_wallpaper_lockscreen">Hintergrundbild und Sperrbildschirm</string>
 
     <!-- settings. -->
-    <string name="settings_category_basic">Basisch</string>
-    <string name="settings_title_back_to_top">Drücken Sie den Zurückknopf um nach oben zurückzuscrollen</string>
+    <string name="settings_category_basic">Haupteinstellungen</string>
+    <string name="settings_title_back_to_top">Drücken Sie den "Zurück" Knopf um nach oben zurückzuscrollen</string>
     <string name="settings_title_custom_api_key">Eigener API Schlüssel</string>
     <string name="settings_summary_custom_api_key">Legen Sie Ihren eigenen API Key und Geheimnis fest.</string>
     <string name="settings_title_language">Sprache</string>
 
     <string name="settings_category_filter">Filter</string>
-    <string name="settings_title_default_order">Standartordnung</string>
-    <string name="settings_title_default_collection_type">Standartkollektionstyp</string>
+    <string name="settings_title_default_order">Standardordnung</string>
+    <string name="settings_title_default_collection_type">Standardkollektionstyp</string>
 
     <string name="settings_category_download">Herunterladen</string>
     <string name="settings_title_download_scale">Skalierung zum Herunterladen</string>
```

Comment:
I doubt you can download a scale here. Funnily scale can also mean "Waage" or so…
Maybe "Skaliert herunterladen" (= "in einer bestimmten Skalierung")?

Or, if there are multiple options connected with this option: "Skalierung beim Herunterladen"